packages/neo-one-monitor/src/types.js

```diff
@@ -1,5 +1,7 @@
 /* @flow */
 import type { Context } from 'koa';
+import type { MetricCollection } from './BrowserMetricsFactory';
```

Comment:
Same thing here, if `MetricCollection` and `CollectingLoggerLogOptions` are part of the public api of `Monitor`, then they should be defined in this file, not somewhere else.

Reply:
I.e. this file should not import from any other file.